docs/sources/installation/upgrading.md

```diff
@@ -277,3 +277,22 @@ For existing alert notification channels, there is no automatic migration of sto
 > Please note that when migrating a notification channel and later downgrading Grafana to an earlier version, the notification channel will not be able to read stored sensitive settings and, as a result, not function as expected.
 
 For provisioning of alert notification channels, refer to [Alert notification channels]({{< relref "../administration/provisioning.md#alert-notification-channels" >}}).
+
+## Upgrading to v7.3
+
+### AWS CloudWatch data source
+
+The AWS CloudWatch data source's authentication scheme has changed in Grafana 7.3. Most importantly the authentication method _ARN_ has been removed, and a new one has been added: _AWS SDK Default_. Existing data source configurations using the former will fall back to the latter. Assuming an IAM role will still work though, and the old _ARN_ method would use the default AWS SDK authentication method under the hood anyway.
+
+Since _ARN_ has been removed as an authentication method, we have instead made it into an option for providing the ARN of an IAM role to assume. This works independently of hte authentication method you choose.
+
+The new authentication method, _AWS SDK Default_, uses the default AWS Go SDK credential chain, which at the time of writing looks for credentials in the following order: 
+
+1. Environment variables.
+1. Shared credentials file.
+1. If your application uses an ECS task definition or RunTask API operation, IAM role for tasks.
+1. If your application is running on an Amazon EC2 instance, IAM role for Amazon EC2.
```

Comment:
Maybe would make more sense to move this to the using cloudwatch documentation and link to it from here? At least there should be a link somewhere in here to using the cloudwatch data source documentation.

Reply:
@marefr Do you mind making the change you have in mind and pushing it? I'm not sure exactly how you would do this.

Reply:
I pushed a change, Added a link to the Using CloudWatch in Grafana. Decided to not add the list of fallbacks to the Using CloudWatch in Grafana page since I couldn't find a good place to put it in. We can always improve the docs later.